lib/llvmopencl/ParallelRegion.cc

```diff
@@ -197,17 +197,18 @@ ParallelRegion::chainAfter(ParallelRegion *region)
       tail = region->at(region->size() - 2);
       t = tail->getTerminator();
     }
-  if (t->getNumSuccessors() != 1)
-    {
+#ifndef NDEBUG
```

Comment:
It would be better use a macro that tells if LLVM is built in the debug mode? As NDEBUG is unrelated to the LLVM build here, right?